lib/index.js

```diff
@@ -2,31 +2,10 @@
 
 const isComponent = require('./helpers/is-component');
 const isNullOrUndefined = require('./helpers/is-null-or-undefined');
-
-const VNodeFlags = {
-	Text: 1,
-	HtmlElement: 1 << 1,
-
-	ComponentClass: 1 << 2,
-	ComponentFunction: 1 << 3,
-	ComponentUnknown: 1 << 4,
-
-	HasKeyedChildren: 1 << 5,
-	HasNonKeyedChildren: 1 << 6,
-
-	SvgElement: 1 << 7,
-	MediaElement: 1 << 8,
-	InputElement: 1 << 9,
-	TextareaElement: 1 << 10,
-	SelectElement: 1 << 11,
-	Fragment: 1 << 12,
-	Void: 1 << 13
-};
+const VNodeFlags = require('./flags');
```

Comment:
Instead of doing this we should just add `inferno-vnode-flags` as a dependency and require that package.